util/settings/settings.go

```diff
@@ -1486,7 +1487,13 @@ func (mgr *SettingsManager) InitializeSettings(insecureModeEnabled bool) (*ArgoC
 		if adminAccount.Enabled {
 			now := time.Now().UTC()
 			if adminAccount.PasswordHash == "" {
-				initialPassword := argorand.RandString(initialPasswordLength)
+				randBytes := make([]byte, initialPasswordLength)
+				_, err := rand.Read(randBytes)
+				if err != nil {
+					return err
+				}
+				initialPassword := hex.EncodeToString(randBytes)
```

Comment:
Due to hex encoding the output string length does match to `initialPasswordLength` value. We need some other way to covert random bytes to string.